### ISSUE TITLE
Update Polkadot Node pkg

### ISSUE THREAD

**Issue by EmilIvanichkovv** [metacraft-labs/nix-blockchain-development#131]
Shall we consider updating the Polkadot node package? There seem to be newer versions available. Additionally, the [repository](https://github.com/paritytech/polkadot) housing the node implementation has been archived in favor of [polkadot-sdk](https://github.com/paritytech/polkadot-sdk/tree/master/polkadot)

**Comment by PetarKirov** [comment]
Yes, we should update our Polkadot package, but perhaps we should also keep the current version, for backwards compatibility.
